instrumentation/sidekiq/lib/opentelemetry/instrumentation/sidekiq/instrumentation.rb

```diff
@@ -38,6 +38,10 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         #   - `:queue` - the span names will be set to '<destination / queue name> <operation>'.
         #   - `:job_class` - the span names will be set to '<job class name> <operation>'.
         option :span_naming,                 default: :queue, validate: %I[job_class queue]
+        # @!macro
+        #   @!method $1
+        #   @instrumentation_option_default `:$2`
```

Comment:
Why using `:$2` here but in `:propagation_style` changed from `:$2` to $2? 
Should line 33 also need to change to $2 if that's the correct format?

Reply:
Somehow I thought it would be easier to see the symbols if they are code blocks.
But it's not my strong request, so it's ok that unified them one way or the other 🙏

Reply:
Ah, I see, it's for yardoc; my bad. It looks good, thanks

Reply:
I don't have any other preference about illustration of boolean and nil value (i.e. could be plain string, code block or darken like `**$2**`); unless others suggest difference.